notebooks/Example_Notebook.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Please add axis labels, and a title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>